components/ui/button/types.tsx

```diff
@@ -0,0 +1,15 @@
+export type ButtonSizes = "lg" | "md" | "sm" | "xl" | "xs";
+
+interface ButtonBaseProps {
```

Comment:
Do we need loading props to handle a loading which is triggered by button clicked? So there will be a spinner icon and the button will be disabled.

Here is the [reference](https://ant.design/components/button/#components-button-demo-loading) I see.

Reply:
Yes, we should. I'll add it to the todo.